trendyqc/trend_monitoring/management/commands/utils/_notifications.py

```
@@ -44,8 +42,8 @@ def slack_notify(message) -> None:
         )
 
 
-def build_report(header: str, final_msg: str, dict_info: dict = {}):
-    """ Given all the messages that a MultiQC report object possesses, create a
+def build_report_for_slack(header: str, final_msg: str, dict_info: dict = {}):
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)